meteor/server/api/ingest/rundownInput.ts

```diff
@@ -426,6 +504,86 @@ export async function regenerateRundown(
 	)
 }
 
+export async function handleUpdatedRundownMetaDataInner(
+	cache: CacheForIngest,
+	ingestRundown: LocalIngestRundown,
+	peripheralDevice?: PeripheralDevice // TODO - to cache?
+): Promise<CommitIngestData | null> {
+	if (!canRundownBeUpdated(cache.Rundown.doc, false)) return null
+	const existingRundown = cache.Rundown.doc
+	if (!existingRundown) {
+		throw new Meteor.Error(404, `Rundown "${ingestRundown.externalId}" does not exist`)
+	}
+
+	const span = profiler.startSpan('ingest.rundownInput.handleUpdatedRundownMetaDataInner')
+
+	logger.info(`Updating rundown ${cache.RundownId}`)
+
+	const extendedIngestRundown = extendIngestRundownCore(ingestRundown, cache.Rundown.doc)
+
+	const selectShowStyleContext = new StudioUserContext(
+		{
+			name: 'selectShowStyleVariant',
+			identifier: `studioId=${cache.Studio.doc._id},rundownId=${cache.RundownId},ingestRundownId=${cache.RundownExternalId}`,
+			tempSendUserNotesIntoBlackHole: true,
+		},
+		cache.Studio.doc
+	)
+	// TODO-CONTEXT save any user notes from selectShowStyleContext
+	const showStyle = await selectShowStyleVariant(selectShowStyleContext, extendedIngestRundown)
+	if (!showStyle) {
+		logger.debug('Blueprint rejected the rundown')
+		throw new Meteor.Error(501, 'Blueprint rejected the rundown')
+	}
+
+	const pAllRundownWatchedPackages = WatchedPackagesHelper.createForIngest(cache, undefined)
+
+	const showStyleBlueprint = await loadShowStyleBlueprint(showStyle.base)
+	const allRundownWatchedPackages = await pAllRundownWatchedPackages
+
+	// Call blueprints, get rundown
+	const { dbRundownData, rundownRes } = await getRundownFromIngestData(
+		cache,
+		ingestRundown,
+		peripheralDevice,
+		showStyle,
+		showStyleBlueprint,
+		allRundownWatchedPackages
+	)
+
+	// Save rundown and baseline
+	const dbRundown = await saveChangesForRundown(cache, dbRundownData, rundownRes, showStyle)
+
+	let segmentChanges: UpdateSegmentsResult | undefined
+	let removedSegments: DBSegment[] | undefined
+	if (!_.isEqual(existingRundown.metaData, dbRundown.metaData)) {
```

Comment:
I think this should be done as `if (!_.isEqual(rundownToSegmentRundown(existingRundown), rundownToSegmentRundown(dbRundown))) {`
so that when more properties are added to `
```suggestion
	if (!_.isEqual(existingRundown.metaData, dbRundown.metaData)) {
```